### PR TITLE
Design update 2

### PR DESCRIPTION
**Pull request by petesfrench**
### Done

- Updates all other notification boxes to match the notification style in the [design](https://www.figma.com/file/ccZsw94LHVpSC8UK3SPnQq/Ubuntu-summit?node-id=37%3A1489)
- Updates buttons to be white, unless there is a reason to have it green (submissions/confirmations/etc)
- Update spacing on the surveys

### QA

To QA you will need the following pages populated:
- At least 2 surveys (one can be blank) to access the survey selection page
- One survey containing each variety of input and a topic with description
- Registration open
- Cfp open

### Fixes

https://github.com/canonical/web-squad/issues/5903

### CODE REVIEW

File: indico_themes_canonical/static/css/ubuntu-summit/call-for-abstracts.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/* ---------- Style for Call to Abstracts/Propoasals page ---------- */
```
Comment (carkod) [inline indico_themes_canonical/static/css/ubuntu-summit/call-for-abstracts.css:1]: Typo `Propoasals`

File: indico_themes_canonical/static/css/ubuntu-summit/header.css [hunk 2]
```diff
@@ -47,22 +35,62 @@ Wraps around the title box
...
-  width: auto;
+  width: var(--full-width);
 }
 
 /*
 The style for the title text.
 */
-.conference-title-link {
-  font-size: 2.228rem;
+.confTitle {
```
Comment (carkod) [inline indico_themes_canonical/static/css/ubuntu-summit/header.css:48]: Are these overrides of existing classes? the usual naming convention would be `conf-title`?

File: indico_themes_canonical/static/css/ubuntu-summit/header.css [hunk 3]
```diff
@@ -47,22 +35,62 @@ Wraps around the title box
...
+  display: inline-block;
+  margin-left: var(--global-margin-left);
+  width: 100%;
+}
+
+.headerLogoCont {
```
Comment (carkod) [inline indico_themes_canonical/static/css/ubuntu-summit/header.css:54]: Same here, could this not be `header-logo-cont`?

**Review by carkod — APPROVED** [review]
LGTM. 

Maybe unrelated but:
- The Help link is overlaping the logo part in the footer in smaller screens
footer-link

The caret in the dropdown looks quite stuck in the edge? But I guess the is partly due to lack of Vanilla.
image

File: indico_themes_canonical/static/css/ubuntu-summit/header.css (hunk 2 above)
Comment (petesfrench) [reply indico_themes_canonical/static/css/ubuntu-summit/header.css]: Exactly this, these are to override previous classes

File: indico_themes_canonical/static/css/ubuntu-summit/header.css (hunk 3 above)
Comment (petesfrench) [reply indico_themes_canonical/static/css/ubuntu-summit/header.css]: Same here, previous classes

**Comment by petesfrench** [comment]
@carkod I have addressed the issue with the footer. As we want to push this live ASAP, I will leave the caret for now and we can re address it in the future if necessary.

**Pull request merged**